terminus-ui/login-form/src/login-form.component.md

```diff
@@ -0,0 +1,34 @@
+<h1>Login Form</h1>
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Basic Usage](#basic-usage)
+- [Customizations](#customizations)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Basic Usage
+
+Create a basic login form with submit function:
+
+```html
+<ts-login-form
+  (submission)="mySumbitFunction($event)"
+></ts-login-form>
+```
+
+
+## Customizations
```

Comment:
I don't think we need to call out all the inputs in this doc. The usage docs are more 'use-case docs'.